server/pkg/webapp/serve.go

```diff
@@ -148,7 +151,67 @@ func serveConfig(r chi.Router, config webappConfig) {
 	r.Get(options.CleanBase(config.appUrl, "custom.css"), func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm not sure I'm a big fan of us re-evaluating the SASS for each request to that custom CSS endpoint.

What I'd do is I'd create a small service (either in `pkg` or the same service we're using to manage custom SCSS) which would be responsible for re-computing the SASS as well as providing it on demand.

This would then solve 2 problems:

1. re-computing sass - CSS for each request
2. moving the evaluation logic out of here (because IMO, it doesn't belong here -- `pkg/webapp` handles the _simple logistics_ to serve webapps to the users; it shouldn't do such _compile time_ heavy lifting).